doc/examples/aggregated_distances.ipynb

```diff
@@ -0,0 +1,654 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "for t in range(h.max_t, h.max_t+1):\n",
+    "    ax = pyabc.visualization.plot_kde_2d(*h.get_distribution(m=0, t=t),\n",
+    "                                         \"theta1\", \"theta2\",\n",
+    "                xmin=0, xmax=1, numx=300,\n",
+    "                ymin=0, ymax=1, numy=300)\n",
+    "    ax.scatter([theta1_true], [theta2_true],\n",
+    "                color=\"C1\",\n",
+    "                label='$\\Theta$ true = {:.3f}, {:.3f}'.format(\n",
+    "                    theta1_true, theta2_true))\n",
+    "    ax.set_title(\"Posterior t={}\".format(t))\n",
+    "    ax.legend()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'theta1': 0.007383515410820385, 'theta2': 0.5690987953623494} 2.3264950476650377 0.1499999999999999\n",
+      "{'theta1': 0.9365720007866767, 'theta2': 0.6538296514399742} 3.348725978941702 0.27\n",
+      "{'theta1': 0.7618780643515053, 'theta2': 0.038030373443562304} 6.358125265938309 0.26\n",
+      "{'theta1': 0.5602929833910603, 'theta2': 0.5921740108348104} 2.2478306798490544 0.24\n",
+      "{'theta1': 0.09558643987758053, 'theta2': 0.1304977681003756} 0.34565544561657124 0.09999999999999998\n",
+      "{'theta1': 0.8676763749565289, 'theta2': 0.828504127993412} 2.611197016602376 0.36999999999999994\n",
+      "{'theta1': 0.4292079923598675, 'theta2': 0.6132687889012809} 1.5035817842518897 0.12\n",
+      "{'theta1': 0.32503069717496, 'theta2': 0.5821984400419155} 0.950811969800647 0.15000000000000008\n",
+      "{'theta1': 0.9269531337922728, 'theta2': 0.4018369575777031} 4.335267675857746 0.4000000000000001\n",
+      "{'theta1': 0.574942476350519, 'theta2': 0.5605028742014329} 2.4344988300943373 0.24\n",
+      "{'theta1': 0.0820849986238988, 'theta2': 0.1353352832366127} 0.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "for _ in range(10):\n",
+    "    p = dict(parameter_prior.rvs())\n",
+    "    sim = model(p)\n",
+    "    print(p, cr_l1_distance(sim, observation), gauss_ks_distance(sim, observation))\n",
```

Comment:
- can you instead generate a plot? it would be easier for interpretation
- or a pandas dataframe with column names?
if not, then I would include and additional printed line before the last print statement as follows:
`print("True parameters:")`